src/config/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@code-climbers/config",
+  "version": "1.0.0",
+  "description": "This is a configuration microservice accessible via websocket",
+  "keywords": [
+    "websocket",
+    "configuration",
+    "flagsmith",
+    "codeclimbers"
+  ],
+  "license": "MIT",
+  "main": "index.js",
+  "module": "module",
+  "author": {
+    "name": "Björn 'Idrinth' Büttner",
+    "url": "https://github.com/idrinth"
+  },
+  "scripts": {
+    "start": "node index.js",
```

Comment:
What will be managing this websocket server when the CLI is distributed? Will oclif start and manage both as a systemd?

Reply:
that would be the intention, yes